distribution/src/main/release/samples/performance/jaxrs/src/main/java/org/apache/cxf/performance/https/client/Client.java

```diff
@@ -85,13 +85,15 @@ public void doJob(WebClient webClient) {
                     //GET
                     Response respGet = webClient.get();
                     Asserts.check(respGet.getStatus() == 200, "Get should have been OK");
+                    respGet.close();
                     break;
                 case 1:
                     //POST
                     Customer customerPost = new Customer();
                     customerPost.setName("Jack");
                     try (Response respPost = webClient.post(customerPost)) {
                         Asserts.check(respPost.getStatus() == 200, "Post should have been OK");
+                        respPost.close();
```

Comment:
@jgoodyear you don't need that, `try (Response respPost = webClient.post(customerPost)) {` closes the response (it is `AutoCloseable`)

Reply:
I'm happy to remove from the POST call - the GET call certainly has lingering connections

Reply:
```
try (Response respGet = webClient.get()) {
     Asserts.check(respGet.getStatus() == 200, "Get should have been OK");
}
```

Reply:
I'll give it a try and reply here.